app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+
+  validates_presence_of :name, :street_address, :city, :state, :zip_code
+  validates_presence_of :password, require: true
+  validates :email, uniqueness: true, presence: true
+
+  validates_confirmation_of :password, :message => "Passwords should match"
+
+  belongs_to :merchant, optional: true
+
+  has_secure_password
+
+  enum role: %w(default merchant_employee admin)
```

Comment:
gives us access to boolean methods on user (user.default? ...) needed for authorization.